### PR TITLE
Mention that we are an open source project based in a university #1184

### PR DESCRIPTION
**Pull request by jamos-tay**
Fixes PowerPointLabs/PowerPointLabs#1184

Updated Description.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -60,7 +60,7 @@
 			Your slides don't have to be this way, and it doesn't take a whole lot of effort to make them better.
 		</p>
 		<p>
-			PowerPointLabs makes creating engaging PowerPoint presentations easy. <strong>Check out what it can do for you below.</strong>
+			PowerPointLabs is an open source project based in the National University of Singapore. It makes creating engaging PowerPoint presentations easy. <strong>Check out what it can do for you below.</strong>
```
Comment (damithc) [inline index.html]: Bold `open source project` and `National University of Singapore`? Can link each too (first one to github and second one to SoC website)

Comment (jamos-tay) [reply index.html]: Sure.

Here's a screenshot:
image

File: index.html [hunk 2]
```diff
@@ -60,7 +60,7 @@
 			Your slides don't have to be this way, and it doesn't take a whole lot of effort to make them better.
 		</p>
 		<p>
-			PowerPointLabs makes creating engaging PowerPoint presentations easy. <strong>Check out what it can do for you below.</strong>
+			PowerPointLabs is an <a href="https://github.com/PowerPointLabs/PowerPointLabs"><strong>open source project</strong></a> based in the <a href="http://www.comp.nus.edu.sg/">National University of Singapore</a>. It makes creating engaging PowerPoint presentations easy. <strong>Check out what it can do for you below.</strong>
```
Comment (Skaty) [inline index.html]: It would be better if we can link to HTTPS versions of the websites if possible.

**Review by Skaty — CHANGES_REQUESTED** [review]
Minor nitpick.

File: index.html (hunk 2 above)
Comment (jamos-tay) [reply index.html]: Thanks, done

**Review by Skaty — APPROVED** [review]
LGTM.

**Pull request merged**